File: BackendAPI/Application/DTOs/Output/KwekerOutputDto.cs
namespace Application.DTOs.Output;

public class KwekerOutputDto
{
    public Domain.Enums.AccountType AccountType { get; init; } = Domain.Enums.AccountType.Kweker;
    public required string Email { get; set; }
    public required string KvkNumber { get; set; }
    public required string CompanyName { get; set; }
    public required string FirstName { get; set; }
    public required string LastName { get; set; }
    public required string Telephone { get; set; }
    public required AddressOutputDto Address { get; set; }
}

File: FrontendWeb/src/declarations/dtos/output/KwekerOutputDto.ts
import { AccountType } from '../../enums/AccountTypes';
import { AddressOutputDto } from './AddressOutputDto';

// KwekerOutputDto.ts
export interface KwekerOutputDto {
	accountType: AccountType;
	email: string;
	kvkNumber: string;
	companyName: string;
	firstName: string;
	lastName: string;
	telephone: string;
	address: AddressOutputDto;
}
